docker/supervisor-distrepos.conf

```
@@ -0,0 +1,9 @@
+[program:distrepos]
+command=/bin/bash -c "/bin/distrepos --action rsync; /bin/distrepos --action mirror; sleep 360"
+autorestart=true
+
+# Log the output of distrepos to supervisord's stdout/err so k8s logging picks it up
+stdout_logfile=/dev/fd/1
+stdout_logfile_maxbytes=0
+stderr_logfile=/dev/stderr
+stderr_logfile_maxbytes=0
```

```suggestion
stderr_logfile_maxbytes=0

```
missing trailing newline here